crawler/pom.xml

```diff
@@ -27,6 +27,7 @@ the License.
 	<artifactId>cas-crawler</artifactId>
 	<name>Catalog and Archive Crawling Framework</name>
 	<description>The Catalog and Archive Service Crawling Framework.</description>
+	<packaging>bundle</packaging>
```

Comment:
maybe we shouldn't change the default packaging per project, but instead create an oodt-bundle that is packaged as an OSGI bundle that combines the constituent sub projects.